docs/docs/dev_docs/tutorials/writing_private_voting_contract.md

```diff
@@ -0,0 +1,251 @@
+import Image from '@theme/IdealImage';
+
+# Writing a private voting smart contract in Aztec.nr
+
+In this tutorial we will go through writing a very simple private voting smart contract in Aztec.nr. You will learn about private functions, public functions, composability between them, and state management.
+
+We will build this:
+
+<Image img={require('/img/tutorials/voting_flow.png')} />
+
+* The contract will be initiated with an admin, stored publicly
+* A voter can vote privately, which will call a public function and update the votes publically. This keeps who they are, what they have voted for, and whether or not they have voted private - but the vote itself public
+* The admin can end the voting period, which is a public boolean
+
+To keep things simple, we won't create ballots or allow for delegate voting. 
+
+## Prerequisites
+
+- You have followed the [quickstart](../getting_started/quickstart.md)
+- Running Aztec Sandbox
+
+## Set up a project
+
+First, create a new contract project with [`nargo`](../getting_started/aztecnr-getting-started.md). 
```

Comment:
Is this the proper link? It says nargo, but then links to aztec.nr.

Reply:
it explains the set up - made the link take the whole sentence instead of nargo. does this make more sense?